src/app/app.component.scss

```
@@ -0,0 +1,82 @@
+.header {
+  background-color: #4b565e;
+  height: 80px;
+  align-items: center;
+  color: white;
+  justify-content: space-between;
+  display: flex;
+  padding: 0 240px;
+}
+.left {
+  display: flex;
+  align-items: center;
+}
+.logo {
+  margin-right: 20px;
+}
+.nav {
+  display: flex;
+}
+.nav-item {
+  margin-right: 20px;
+}
+.right {
+  display: flex;
+  align-items: center;
+}
+
+.logo-phone {
+  margin-right: 15px;
+}
+.logo-like {
+  margin-right: 15px;
+}
+.button {
+  border: 1px solid yellow;
+  border-radius: 20px;
+  padding: 5px 15px;
+  background-color: transparent;
+  color: white;
+  cursor: pointer;
+  align-items: center;
+  display: flex;
+}
+.top-bar {
+  background-color: #f0ba4e;
+  height: 48px;
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+  padding: 0px 240px;
```

```suggestion
  padding-inline:240px;
```

Если верхний/нижний отступы не нужны